package.json

```diff
@@ -16,7 +16,7 @@
   "author": "Traceur Authors",
   "license": "Apache License 2.0",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=0.12"
```

Comment:
I think @johnjbarton convinced me. If we work with Node 0.10 and io.js we should not limit this.